guides/RestAPIGuide.md

```diff
@@ -1072,203 +1072,203 @@ The most common attachment type is 'file'.
 }
 ```
 **Link Attachment**
-```
+```json
 {
   * "type" : "url",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "url" : <string>,
-  "disabled" : <boolean>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "url" : "<string>",
+  "disabled" : "<boolean>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Web Page Attachment**
-```
+```json
 {
   * "type" : "htmlpage",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "filename" : <string>,
-  "size" : <int>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "filename" : "<string>",
+  "size" : "<int>",
```

Comment:
Shouldn't need to quote integers.

Reply:
https://github.com/equella/equella.github.io/pull/25#discussion_r234487858